### PR TITLE
update step-36.md

### PR DESCRIPTION
**Pull request by Abdullah-Irfan-web**
changed 'a' elements to 'img' elements

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by Abdullah-Irfan-web** [comment]
okay i have reviewed it on codesee

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/step-036.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-36
 
 # --description--
 
-The `input` element allows you several ways to collect data from a web form. Like anchor (`a`) elements, `input` elements are <dfn>self-closing</dfn> and do not need closing tags.
+The `input` element allows you several ways to collect data from a web form. Like images (`img`) elements, `input` elements are <dfn>self-closing</dfn> and do not need closing tags.
```
Comment (jdwilkin4) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/step-036.md:10]: In the second sentence, we should change images (`img`) elements to just say `img` elements. 

Proposed change: Like `img`elements,

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Comment by jdwilkin4** [comment]
Hi @Abdullah-Irfan-web ! 
Thank you for making these changes.
We recently just merged another PR that fixed this issue so we are going to go ahead and close this PR.
If you are interested in looking for more good first issues, then you can look into this freeCodeCamp repo to contribute to.
https://github.com/freeCodeCamp/Developer_Quiz_Site/issues

Happy coding!

**Comment by Sembauke** [comment]
Hey @Abdullah-Irfan-web,

Thank you for opening this pull-request. Unfortunately this issue has already been fixed by #44842.

This means we will not be accepting your pr.

Thank you for your understanding,
Have a great day and we hope to see more of you in the future, happy coding 😄

**Pull request closed without merging**